From: Alec Leamas <leamas.alec@gmail.com>
Date: Mon, 11 Aug 2025 15:00:31 +0200
Subject: tests: disable file driver tests on distro builders

The file driver tests fails for unknown reasons on some esoteric
Debian platforms like IBM/S390 and mips. Possibly little endian
problems. We are not particularly interested in the file driver
on these platforms, so just disable them in distro builds

Forwarded: not-needed
---
 test/tests.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/tests.cpp b/test/tests.cpp
index b575ec6..086b914 100644
--- a/test/tests.cpp
+++ b/test/tests.cpp
@@ -964,6 +964,7 @@ TEST(Navmsg2000, to_string) {
   EXPECT_EQ(string("n2000-    PGN: 6385516 [  ]"), msg->to_string());
 }
 
+#ifndef OCPN_DISTRO_BUILD
 TEST(FileDriver, Registration) {
   wxLog::SetActiveTarget(&defaultLog);
   auto driver = std::make_unique<FileCommDriver>("test-output.txt");
@@ -990,6 +991,7 @@ TEST(FileDriver, output) {
   ss << f.rdbuf();
   EXPECT_EQ(ss.str(), string("n2000-    PGN: 6385516 [  ]"));
 }
+#endif //  OCPN_DISTRO_BUILD
 
 #if 0
 // FIXME (comm_drv_file, see FIXME there)
